footer.inc

```diff
@@ -14,7 +14,7 @@
 					id="download-button"
 					class="btn-large waves-effect waves-light light blue lighten-1 modal-trigger" href='village_fund_payment_view.php' 
 					>Give Monthly</a>	
-				<script>
```

Comment:
Can we just delete this section?  Did you replace it below?

Reply:
yes. i just left it in for you to review and potentially to steal some of the code, if necessary